Dockerfile

```diff
@@ -0,0 +1,55 @@
+# Build it like this:
+# docker build --tag webportal-service:improve-build .
+
+# Run it like this:
+# docker run -p 80:80 -v /absolute/location/of/your/export.zip:/home/specify/webportal-installer/specify_exports/export.zip webportal-service:improve-build
+
+
+
+FROM ubuntu:20.04 as specify_base_ubuntu
+
+LABEL maintainer="Specify Collections Consortium <github.com/specify>"
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Get Ubuntu packages
+RUN apt-get update && apt-get -y install \
+	curl \
+	default-jre \
+        git \
+	lsof \
+	make \
+	python3 \
+	python3-lxml \
+	unzip \
+	wget && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+
+FROM specify_base_ubuntu as webportal
+
+# Get Web Portal
+RUN cd /home/ \
+	&& mkdir specify \
+	&& cd specify \
+	&& git clone https://github.com/cryoarks/webportal-installer/
```

Comment:
I think this git clone should be replaced with a COPY directive so that the container ends up with the version currently checked out in the working directory.

Reply:
This is a cool idea. Thanks for providing an explanation.

Reply:
Yes, I agree. A much better way of doing it.